src/main/java/org/broadinstitute/hellbender/engine/filters/AmbiguousBaseReadFilter.java

```diff
@@ -7,38 +7,42 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Filters out reads with more than a threshold number of N's
+ * Filters out reads with more than a threshold number of N's. By default will determine the threshold by the given
+ * maximum fraction of bases in the read. This can be overridden by setting the maximum number of ambiguous bases instead.
  */
 @DocumentedFeature(groupName= HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
 public final class AmbiguousBaseReadFilter extends ReadFilter {
 
     private static final long serialVersionUID = 1L;
+    private final Integer maxAmbiguousBases;
```

Comment:
I think that this should be exposed to the user in the command line, because if not it could rise unexpected behavior. Imagine that a default filter is applied using the constructor with maxAmbigBases, and the user provides an `--ambigFilterFrac` argument to the CLI. The fraction specified by the user is ignored without noticing. To expose the argument as mutually-exclusive with the fraction, something like this should be done:

```java
@Argument(doc = "Maximum number of ambiguous bases. If null, use the fraction of ambiguous bases; otherwise, ignores the fraction", fullName = "maxAmbigBases", optional = true, mutex = {"ambigFilterFrac})
public final Integer maxAmbiguousBases = null;
```

It also needs to update the `@Argument` for the maxAmbiguousBaseFraction by adding `mutex = {"maxAmbigBases"}`.